components/Menu/Menu.tsx

```diff
@@ -61,56 +75,39 @@ export default function Menu() {
   }
 
   return (
-    <div className={`menu`}>
-      <Logo bigSize={false} />
+    <div className="menu">
+      <h1 className={clsx(josefin.className, "menu__logo")}>NOTQE</h1>
       {isMobileSize && isUserLogIn && (
         <RxHamburgerMenu
-          className={"menu__hamburger-menu"}
+          className="menu__hamburger-menu"
           color={darkModeTheme ? "#eeeee1" : "#3e4247"}
           onClick={() => {
             setShowMenu(!showmenu);
           }}

```

Comment:
To lepiej wyjąć poza JSX i zrobić toggleShowMenu = () => setShowMenu(show => !show);